keepassxc-browser/_locales/en/messages.json

```diff
@@ -0,0 +1,742 @@
+    },
+    "optionsCustomFieldsNotFound": {
+        "message": "No saved custom credential fields found.",
+        "description": "Shown when no saved custom credentials are saved."
+    },
+    "optionsCustomFieldsTabHelpTextFirst": {
+        "message": "If KeePassXC-Browser detects the wrong credential fields, you are able to specify the correct fields by yourself.",
+        "description": "Saved custom credential fields info text, first part."
+    },
+    "optionsCustomFieldsTabHelpTextSecond": {
+        "message": "Go to the page and click on the KeePassXC-Browser icon, then select ",
+        "description": "Saved custom credential fields info text, second part."
+    },
+    "optionsCustomFieldsTabHelpTextThird": {
+        "message": "On this page you can manage saved custom credential fields.",
+        "description": "Saved custom credential fields info text, third part."
+    },
+    "optionsCustomFieldsRemove": {
+        "message": "Remove saved custom credential fields?",
+        "description": "Confirmation text when removing saved custom credential fields."
+    },
+    "optionsCustomFieldsConfirmation": {
+        "message": "Do you really want to remove the saved custom credential fields on the page: $1",
+        "description": "Confirmation text when removing saved custom credential fields."
+    },
+    "optionsCustomFieldsConfirmationHelpText": {
+        "message": "KeePassXC-Browser will automatically detect the credential fields next time you visit this page.",
+        "description": "Part of confirmation text when removing saved custom credential fields."
+    },
+    "optionsSitePreferencesNotFound": {
+        "message": "No ignored sites found.",
+        "description": "Shown when no ignored sites are saved."
+    },
+    "optionsSitePreferencesRemove": {
+        "message": "Remove site?",
+        "description": "Confirmation text when removing ignored sites from the list."
+    },
+    "optionsSitePreferencesTabHelpTextFirst": {
+        "message": "Sites on this page have special handling methods associated with them.",
+        "description": "Site preferences info text, first part."
+    },
+    "optionsSitePreferencesTabHelpTextSecond": {
+        "message": "To ignore new/modified credentials on a specific site, add it below or click the blinking KeePassXC-Browser icon and select",
+        "description": "Site preferences info text, second part."
+    },
+    "optionsSitePreferencesTabHelpTextThird": {
+        "message": "If a site is fully ignored (<em>Disable all features</em> is selected), then the plugin will do nothing when visiting that site.",
+        "description": "Site preferences info text, third part."
+    },
+    "optionsSitePreferencesTabHelpTextFourth": {
+        "message": "Enabling the <em>Username-Only Detection</em> feature allows KeePassXC-Browser to fill-in pages on sites that do present a separated username and password input.",
```

Comment:
Username-only detection allows KeePassXC-Browser to fill in login details on websites with separate pages for  username and password.